app/presenters/formats/edition_format_presenter.rb

```diff
@@ -37,7 +41,7 @@ def required_fields(republish)
         need_ids: [],
         public_updated_at: public_updated_at.to_datetime.rfc3339(3),
         publishing_app: "publisher",
-        rendering_app: "frontend",
+        rendering_app: RENDERING_APP.fetch(edition.format.try(:to_sym), "frontend"),
```

Comment:
I'd like to see this functionality rewritten as a ['default' value in this base class](https://github.com/alphagov/publisher/blob/master/app/presenters/formats/edition_format_presenter.rb#L54), and overridden in the [child class](https://github.com/alphagov/publisher/blob/master/app/presenters/formats/answer_presenter.rb) as appropriate. 

I'm happy to pair on it with you if that will help.